cinema/views.py

```diff
@@ -21,22 +22,22 @@
 )
 
 
-class GenreViewSet(viewsets.ModelViewSet):
+class GenreListCreateView(generics.ListCreateAPIView):
```

Comment:
You must use viewsets
<img width="626" alt="image" src="https://github.com/user-attachments/assets/df0ea14c-fcbf-4577-a68c-d1eee2c6e369">